### PR TITLE
da: translate a number of tiny/trivial messages

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

File: po/da.po [hunk 1]
```diff
@@ -10138,83 +10180,83 @@ msgstr ""
...
 
 #: src/bare-metal/no_std.md:32
 msgid "`fence`, `AtomicBool`, `AtomicPtr`, `AtomicU32`..."
-msgstr ""
+msgstr "`fence`, `AtomicBool`, `AtomicPtr`, `AtomicU32`..."
 
 #: src/bare-metal/no_std.md:33
 msgid "`Duration`"
-msgstr ""
+msgstr "`Duration`"
```
Comment (thedataking) [inline po/da.po:10211]: Varighed?

**Review by thedataking — APPROVED** [review]
LGTM

File: po/da.po (hunk 1 above)
Comment (mgeisler) [reply po/da.po:10211]: This is a Rust type, so it should not be translated.

File: po/da.po [hunk 2]
```diff
@@ -6177,51 +6177,51 @@ msgstr ""
...
 
 #: src/std/hashmap.md:10
 msgid "\"Adventures of Huckleberry Finn\""
-msgstr ""
+msgstr "\"Huckleberry Finn\""
 
 #: src/std/hashmap.md:11
 msgid "\"Grimms' Fairy Tales\""
-msgstr ""
+msgstr "\"Grimms eventyr\""
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:6192]: Yeah, I was wondering about that myself :slightly_smiling_face: I took the case directly from Wikipedia: https://da.wikipedia.org/wiki/Grimms_eventyr — the same for the other books which have Danish titles. So I'll keep it like this for now until someone comes by with a picture of the actual book :smile:

File: po/da.po [hunk 3]
```diff
@@ -5666,11 +5666,11 @@ msgstr ""
 
 #: src/methods/example.md:33
 msgid "\"Monaco Grand Prix\""
-msgstr ""
+msgstr "\"Monacos Grand Prix\""
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:5669]: Right! This is also directly from Wikipedia: https://da.wikipedia.org/wiki/Monacos_Grand_Prix.

File: po/da.po [hunk 4]
```diff
@@ -4267,7 +4267,7 @@ msgstr ""
 
 #: src/pattern-matching/match-guards.md:12
 msgid "\"These are twins\""
-msgstr ""
+msgstr "\"De er tvillinger\""
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:4270]: Thanks!

```suggestion
msgstr "\"Disse er tvillinger\""
```

File: po/da.po [hunk 5]
```diff
@@ -3537,30 +3537,30 @@ msgstr ""
...
 #: src/basic-syntax/static-and-const.md:71
 #: src/basic-syntax/static-and-const.md:72
 msgid "Yes"
-msgstr ""
+msgstr "Ja"
 
 #: src/basic-syntax/static-and-const.md:68
 msgid "No (inlined)"
-msgstr ""
+msgstr "Nej (indlejret)"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:3563]: Good point, perhaps something like this:

```suggestion
msgstr "Nej (inlinet)"
```

[the remark this answers is not in the document]
Comment (thedataking) [reply po/da.po:3563]: agreed that "inlinet" is better here.

File: po/da.po (hunk 2 above)
[the remark this answers is not in the document]
Comment (thedataking) [reply po/da.po:6192]: https://dengodebog.dk/vare/grimms-eventyr-ved-anine-rud/ 😸 I cherry picked this one but truth be told, a quick google search revealed a lot of variability so I rescind all nitpicks related to casing of book titles.

File: po/da.po [hunk 6]
```diff
@@ -3170,12 +3170,12 @@ msgstr ""
 #: src/exercises/day-1/for-loops.md:73
 #: src/exercises/day-1/solutions-morning.md:49
 msgid "\"matrix:\""
-msgstr ""
+msgstr "\"matrix:\""
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:3173]: This is the name of a variable used in the exercise.

**Review by thedataking — APPROVED** [review]

File: po/da.po [hunk 7]
```diff
@@ -3570,23 +3570,23 @@ msgstr ""
...
 
 #: src/basic-syntax/static-and-const.md:70
 msgid "Yes (unsafe)"
-msgstr ""
+msgstr "Ja (_unsafe_)"
 
 #: src/basic-syntax/static-and-const.md:71
 msgid "Evaluated at compile time"
-msgstr ""
+msgstr "Evalueret ved kompilering"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:3585]:
```suggestion
msgstr "Evalueret ved kompileringstid"
```

File: po/da.po [hunk 8]
```diff
@@ -3570,23 +3570,23 @@ msgstr ""
...
 
 #: src/basic-syntax/static-and-const.md:71
 msgid "Yes (initialised at compile time)"
-msgstr ""
+msgstr "Ja (initialiseret ved kompilering)"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:3589]:
```suggestion
msgstr "Ja (initialiseret ved kompileringstid)"
```

File: po/da.po [hunk 9]
```diff
@@ -4178,15 +4178,15 @@ msgstr ""
...
 
 #: src/pattern-matching/destructuring-structs.md:16
 msgid "\"y = 2, x = {i:?}\""
-msgstr ""
+msgstr "\"y = 2, x = {i:?}\""
 
 #: src/pattern-matching/destructuring-structs.md:17
 msgid "\"y = {y}, other fields were ignored\""
-msgstr ""
+msgstr "\"y = {y}, andre felter er ignoreret\""
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:4189]: Thanks! I guess it doesn't matter much, but it's fun for me to try and make a nice translation :slightly_smiling_face: 

```suggestion
msgstr "\"y = {y}, andre felter blev ignoreret\""
```

**Pull request merged**